plugins/woocommerce/tests/php/src/Internal/AbilitiesApi/AbilitiesApiIntegrationTest.php

```diff
@@ -321,16 +380,25 @@ function () use ( $ability_id ) {
 	 * @group abilities-api
 	 */
 	public function test_rest_endpoints_are_registered() {
-		// Ensure the bootstrap file has been loaded by checking for the class.
-		$this->assertTrue( class_exists( 'WP_REST_Abilities_Init' ), 'Bootstrap should load WP_REST_Abilities_Init class' );
+		// Ensure REST API classes are loaded.
+		// WordPress 6.9+ uses core controllers with different namespace, earlier versions use vendor package.
+		$is_wp_69_plus = class_exists( 'WP_Ability_Categories_Registry' );
```

Comment:
Since we use "is WP 6.9+" check more than once, would it be better to extract it to a separate function?